backend/src/test/java/com/bakdata/conquery/integration/tests/deletion/DatasetDeletionTest.java

```diff
@@ -113,7 +113,7 @@ public void execute(String name, TestConquery testConquery) throws Exception {
 
 			conquery.getNamespace().getStorage().getTables().stream()
 					.map(Table::getId)
-					.forEach(conquery.getDatasetsProcessor()::deleteTable);
```

Comment:
Hier auch force=true testen

Reply:
habe es jetzt einfach konsolidiert in nur tables entfernen, das muss dann ja die konzepte mit abreissen